doc/md/integrations/github-actions.mdx

```diff
@@ -218,4 +228,141 @@ be checked out locally or you will see an error such as:
 Atlas failed with code 1: Error: git diff: exit status 128
 ```
 
-The full list of input options can be found in [action.yml](https://github.com/ariga/atlas-action/blob/master/action.yml).
\ No newline at end of file
+The full list of input options can be found in [action.yml](https://github.com/ariga/atlas-action/blob/master/action.yml).
+
+## `ariga/atlas-deploy-action`
+
+You can use [ariga/atlas-deploy-action](https://github.com/ariga/atlas-deploy-action) to deploy migrations to your database directly from
+GitHub Actions.
+
+:::info
+
+Atlas needs network access to your database to deploy migrations,
+so make sure your database is either publicly accessible or that you have otherwise enabled
+network access to it from your GitHub Actions runners.
+
+:::
+
+This action supports two workflows:
+
+- *Local* - the migration directory is checked in to the repository.
+- *Cloud* - the migration directory is [connected to Atlas Cloud](https://atlasgo.io/cloud/directories).
+Runs are reported to your Atlas Cloud account.
+
+### Examples
+
+
+Notice that the following examples rely on a `DATABASE_URL` secret being set in your repository.
+
+To learn how to set secrets, read [GitHub's documentation](https://docs.github.com/en/actions/reference/encrypted-secrets).
+
+The `DATABASE_URL` secret should be set to the URL of your database, for examples please see [Atlas URL formats](/concepts/url).
+
+#### Local
+
+```yaml
+name: Deploy Database Migrations
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Deploy Atlas Migrations
+        uses: ariga/atlas-deploy-action@v0
+        with:
+          url: ${{ secrets.DATABASE_URL }}
+          dir: path/to/migrations
+```
+
+#### Deploy from Cloud
+
+```yaml
+name: Deploy Database Migrations
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Deploy Atlas Migrations
+        uses: ariga/atlas-deploy-action@v0
+        with:
+          url: ${{ secrets.DATABASE_URL }}
+          cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
+          cloud-dir: hello # replace with your directory name
+```
+
+### Reference
+
+#### Inputs
+
+- `url`: URL to target database (should be passed as a secret). (Required)
+- `dir`: Local path of the migration directory in the repository. (Optional)
+- `cloud-token`: Token for using Atlas Cloud (should be passed as a secret). (Optional)
+- `cloud-dir`: Name of the migration directory in the cloud. (Must be set if `cloud-token` is set)
+- `cloud-tag`: Tag of the migration version in the cloud. (Optional)
```

Comment:
I think flags should have this format (optional/required first), and mention their default. e.g., the `latest` tag.
https://atlasgo.io/declarative/apply